gitchStyle.css

```diff
@@ -0,0 +1,136 @@
+@import url('https://fonts.googleapis.com/css2?family=Mulish&display=swap');
+
+
+html{
+    scroll-behavior: smooth;
+}
+
+body {
+    font-family: 'Mulish', sans-serif;
+    margin: 0;
+}
+
+/* INDEX */
+
+.canvas{
+    height: 100vh;
+    display: grid;
+    grid-template-columns: 1fr 1fr 1fr 1fr 1fr;
+    grid-template-rows: 10vh 1fr;
+    grid-template-areas:
+        "nav nav nav nav nav"
+        "sidebar main main main main";
+    grid-gap: 0.2rem;
+}
+
+.heading {
+    font-size: 2.5rem;
+    width: 100%;
+    position: sticky;
+    top: 11vh;
+    background: white;
+    padding: 1rem 0;
+    margin: 1rem;
+    z-index: 998;
+}
+
+.code-block {
+    margin: 1rem;
+    max-width: 100vw;
+}
+
+/* INSTALLING */
+
+.primary-color {
+    color: var(--primary-color);
+}
+
+
+/* SIDEBAR */
+
+.sidebar{
+    grid-area: sidebar;
+    display: flex;
+    flex-direction: column;
+    justify-content: space-between;
+}
+
+.sidebar-menu {
+    position: sticky;
+    top: 15vh;
+    list-style-type:none;
+    font-size: 1.5rem;
+    display: flex;
+    flex-direction: column;
+    /* justify-content: center; */
+    align-items: center;
+    padding: 2rem 0;
```

Comment:
![image](https://user-images.githubusercontent.com/41542536/117573426-5dac9680-b0f5-11eb-97af-90abb9314382.png)

In the sidebar menu, the list items are getting hidden behind the socials-menu which is position fixed so give the sidebar-menu  ``` height:60vh; overflow-y:auto ```